canonical-websites/www.ubuntu.com
# copy updates to service terms

## Done

Updated the service terms page to reflect the copy doc changes made by legal 

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/legal/service-terms](http://0.0.0.0:8001/legal/service-terms)
- Check page matches [copydoc](https://docs.google.com/document/d/1W0WJrXaIvC54MDPgS62xfgPhqJaSE7RcZ2mjIqz1Y2o/edit?ts=5b084c41)


## Issue / Card

Fixes: #3243
